package.json

```diff
@@ -89,25 +88,24 @@
     "screwdriver-build-bookend": "^2.3.2",
     "screwdriver-cache-bookend": "^1.0.7",
     "screwdriver-command-validator": "^1.0.5",
-    "screwdriver-config-parser": "^4.9.0",
+    "screwdriver-config-parser": "^4.10.0",
     "screwdriver-coverage-bookend": "^1.0.2",
-    "screwdriver-coverage-sonar": "^1.0.13",
-    "screwdriver-data-schema": "^18.42.0",
+    "screwdriver-coverage-sonar": "^1.0.14",
+    "screwdriver-data-schema": "^18.43.3",
     "screwdriver-datastore-sequelize": "^5.6.1",
     "screwdriver-executor-docker": "^4.0.2",
     "screwdriver-executor-k8s": "^13.3.6",
-    "screwdriver-executor-k8s-vm": "^2.8.4",
-    "screwdriver-executor-nomad": "^1.0.13",
```

Comment:
This is breaking our build because `nomad` is not using the newest `executor-base`. Not sure why it was added, since our cluster doesn't use it. Removing.